src/lib/components/TableOfContents/index.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {BaseElement} from '../BaseElement';
+
+/**
+ * Element that renders table of contents.
+ * @extends {BaseElement}
+ * @final
+ */
+class TableOfContents extends BaseElement {
+  static get properties() {
+    return {
+      title: {type: String},
+      titleId: {type: String, attribute: 'title-id'},
+    };
+  }
+
+  constructor() {
+    super();
+    this.close = this.close.bind(this);
+    this.open = this.open.bind(this);
+    this.scrollSpy = this.scrollSpy.bind(this);
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+
+    this.tocInnerDiv = document.createElement('div');
+    this.tocInnerDiv.classList.add('w-toc__content');
+    this.tocInnerDiv.innerHTML = this.innerText;
```

Comment:
The post is `/use-lazysizes-to-lazyload-images`, I hadn't been able to figure out why though...

<img width="1044" alt="Screen Shot 2020-06-11 at 3 54 44 PM" src="https://user-images.githubusercontent.com/11811422/84447078-f09a3380-abfb-11ea-8c3b-fdabafe0c70b.png">

Possibly because there is an `<img>` in a title? => https://web.dev/use-lazysizes-to-lazyload-images/#update-lessimggreater-andor-lesspicturegreater-tags

Reply:
Hold on, may have found a fix...

In `/src/site/_transforms/responsive-images/index.js` I can check if there is an `originalSrc` and if there isn't skip that img element.

<img width="542" alt="Screen Shot 2020-06-11 at 4 01 57 PM" src="https://user-images.githubusercontent.com/11811422/84447470-e6c50000-abfc-11ea-8e42-100176990d4a.png">